about_betydb.md

```diff
@@ -57,13 +60,33 @@ summary, we hope to inspire new scientific perspectives on biofuel crop
 ecology that are based on a comprehensive evaluation of available
 knowledge.
 
-All public data in BETYdb is made available under the [Open Data Commons Attribution License (ODC-By) v1.0](http://opendatacommons.org/licenses/by/1-0/){target="_blank"}. You are free to share, create, and adapt its contents. Data in tables having an an access_level column and in rows where the access_level value is 1 or 2 are not covered by this license but may be available for use with consent.
+All public data in BETYdb is made available under the [Open Data Commons Attribution License (ODC-By) v1.0](http://opendatacommons.org/licenses/by/1-0/){target="_blank"}. You are free to share, create, and adapt its contents. Data in tables having an an access_level column, and rows where the access_level value is 1 or 2 are not covered by this license but may be available for use with consent.
 
-Please cite the source of data as:
+## Citation
+
+If you use or refer to BETYdb data or software in your research, please cite this paper:
+
+> LeBauer, D., Kooper, R., Mulrooney, P., Rohde, S., Wang, D., Long, S. P., & Dietze, M. C. (2018). BETYdb: a yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production. _GCB Bioenergy, 10_, 61–71.  [doi:10.1111/gcbb.12420](https://doi.org/10.1111/gcbb.12420){target="_blank"}
+
+In addition, if you are using _data_ from an instance of BETYdb, please
+
+1. Use the citations associated with each record if the publisher will allow
+2. Archive a copy of the data used in the study. The data in BETYdb is not versioned.  
+2. Refer to the footer of the instance of BETYdb that you are using for citation information.
+   * For example, if you are using data from BETYdb.org, please cite the source of data as:
 
 > LeBauer, David; Dietze, Michael; Kooper, Rob; Long, Steven; Mulrooney, Patrick; Rohde, Gareth Scott; Wang, Dan; (2010): Biofuel Ecophysiological Traits and Yields Database (BETYdb); Energy Biosciences Institute, University of Illinois at Urbana-Champaign. [http://dx.doi.org/10.13012/J8H41PB9](http://dx.doi.org/10.13012/J8H41PB9){target="_blank"}
```

Comment:
@dlebauer  This doesn't match the current footer at www.betydb.org.  It essentially matches the version that was in the footer before we moved to putting this information in a YAML customization file (except that the author name list is more like APA style whereas the former footer's list is more like Chicago style).

For reference, the current footer has
> LeBauer, David, Rob Kooper, Patrick Mulrooney, Scott Rohde, Dan Wang, Stephen Long, Michael Dietze, (2017). BETYdb: A yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production. GCB Bioenergy. doi:10.1111/gcbb.12420

In particular, the current footer has a different date year, different author list, and it references and uses the DOI of the GCB paper.

So I assume the example should be updated to match the current footer at betydb.org.  But first let me know if the citation information currently in the footer is correct or needs to be updated as well.  (As a side note, the contact phone number currently in the footer is your old U. of I. number, so that definitely needs to be updated.)

Reply:
Thanks for catching and fixing this. This is the official citation:

LeBauer, David, Rob Kooper, Patrick Mulrooney, Scott Rohde, Dan Wang, Stephen P. Long, and Michael C. Dietze. "BETYdb: a yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production." GCB Bioenergy 10, no. 1 (2018): 61-71. [doi:10.1111/gcbb.12420](https://doi.org/10.1111/gcbb.12420)

Would be great if you can preserve the doi hyperlink